starboard/android/apk/app/src/main/java/dev/cobalt/media/AudioTrackBridge.java

```diff
@@ -70,6 +70,7 @@ public AudioTrackBridge(
 
     tunnelModeEnabled = tunnelModeAudioSessionId != -1;
     int channelConfig;
+    String sampleTypeString = "ENCODING_INVALID";
```

Comment:
Consider moving this before line 167, and add an empty line before it.  It's usually preferred to declare a variable close to its use.

Reply:
Done.